### ISSUE TITLE
Very Good!!!!

### ISSUE THREAD

**Issue by rodrigosmarques** [alann-maulana/flutter_parse#2]
Congratulations on your library.

Do you intend to implement other objects such as File, Installation?

**Comment by rodrigosmarques** [comment]
Yes. I chose Parse as Backend because of its Framework. Powerful and simple to use.
I'm starting on Flutter. I have seen several packages, but all access via REST and with limitations. These packages do not serve me in my projects.
I had started studying how to make each call via Platform View, but as a newcomer to Flutter I was lost.
When I discovered your package, I cheered up.

My congratulations and I left my star on it.

I did not get tested yet, I'll test and I'm anxious for news.

Note: (My English is not the best, so I had to ask Google for help)
